doc/content/reference/frequency-plans/adding-frequency-plans.md

```diff
@@ -0,0 +1,14 @@
+---
+title: "Adding Frequency Plans"
+description: ""
+---
+
+{{% tts %}} uses frequency plans from the [lorawan-frequency-plans Github repository.](https://github.com/TheThingsNetwork/lorawan-frequency-plans/)
+
+To add frequency plans, you may open a pull request on the [lorawan-frequency-plans Github repository](https://github.com/TheThingsNetwork/lorawan-frequency-plans/), which makes the plan available in most deployments.
```

Comment:
> which makes the plan available in most deployments

This is not completely accurate. Most deployments (including TTI Cloud Hosted) only pull the frequency plans when a new version is deployed. Other deployments use a fork of that repository, which will not update automatically.